### PR TITLE
Update BS4 SCSS Definitions

### PR DESCRIPTION
**Pull request by RyanMulready**
Merge second after merging `main`

### CODE REVIEW

File: es-bs-base/scss/_variables.scss [hunk 1]
```diff
@@ -274,7 +276,7 @@ $embed-responsive-aspect-ratios: join(
 // Font, line-height, and color for body text, headings, and more.
 
 // stylelint-disable value-keyword-case
-$font-family-sans-serif:      "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
+$font-family-sans-serif:      "Inter", "Helvetica Neue", Roboto, "Segoe UI", -apple-system, BlinkMacSystemFont, Arial, "Noto Sans", "Liberation Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
```
Comment (tomleo) [inline es-bs-base/scss/_variables.scss:279]: In the absense of "Inter" wouldn't it be better to fall back to "San Francisco" vs. "Helvetica Neue"? What's the rational for moving `-apple-system` lower in the fallback order?

File: es-bs-base/scss/_variables.scss [hunk 2]
```diff
@@ -384,13 +386,13 @@ $table-border-level:          -6 !default;
...
-$input-btn-padding-x:         .75rem !default;
+$input-btn-padding-y:         0.9rem !default;
+$input-btn-padding-x:         1rem !default;
 $input-btn-font-family:       $font-family-sans-serif !default;
 $input-btn-font-size:         $font-size-base !default;
-$input-btn-line-height:       $line-height-base * 0.9 !default;
+$input-btn-line-height:       1.2 !default;
 
-$input-btn-focus-width:       .2rem !default;
+$input-btn-focus-width:       0 !default;
```
Comment (tomleo) [inline es-bs-base/scss/_variables.scss:395]: Are there any a11y implications to removing addition width on focus? This doesn't affect focus outlines right?

**Review by tomleo — APPROVED** [review]
Assuming the changes I should look at here are only in `_variables.scss` 👍

File: es-bs-base/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@energysage/es-bs-base",
   "description": "Base CSS for EnergySage design system based on Bootstrap",
-  "version": "0.1.9",
+  "version": "0.3.0",
```
[the remark this answers is not in the document]
Comment (RyanMulready) [reply es-bs-base/package.json:4]: There were `0.2.X` versions already published. Felt better just to skip to the cleaner `0.3.X` version

File: es-bs-base/scss/_variables.scss (hunk 1 above)
Comment (RyanMulready) [reply es-bs-base/scss/_variables.scss:279]: I have a design friend who is obsessed with typography. This is the order he recommended, I'm paraphrase but he said "dont put system fonts before web fonts" "Helvetica is a paid version of Roboto, if I paid for a font I want to see it."

That said we can play with the order more I don't feel strongly about this change.

File: es-bs-base/scss/_variables.scss (hunk 2 above)
Comment (RyanMulready) [reply es-bs-base/scss/_variables.scss:395]: Yea I wondered this about a few of these changes regarding focus and a11y. I'm not 100% sure the correct thing but you are right there are still border/color changes for focus.

**Pull request merged**